### PR TITLE
Added new resource to manage lookup table files

### PR DESCRIPTION
**Pull request by akohring**

### CODE REVIEW

**Comment by akohring** [comment]
The acceptance test will require the Splunk App for Lookup File Editing.  What is the best way to add this addon to the test instance?  https://splunkbase.splunk.com/app/1724

**Comment by akohring** [comment]
Please review and re-run the Build and test job.  I added the splunk-app-for-lookup-file-editing app to the acceptance runtime.

**Comment by akohring** [comment]
@rrossetti-splunk Please review this PR.  The last time you ran the Build and Test, master was in a broken state.  I have successfully run the test GH action on my branch.

File: terraform/main.tf
[the remark this answers is not in the document]
Comment (akohring) [reply terraform/main.tf:1]: It is not an example.  This terraform folder is volume mounted into the ACC test container and terraform is applied on the container to install the Splunk lookup app, which is required for the TestAccSplunkLookupTableFile test to pass.

**Review by rrossetti-splunk — APPROVED** [review]

**Pull request merged**